main.py

```diff
@@ -75,6 +78,13 @@ def export_streams_to_clipboard(self):
         clipboard.clear(mode=clipboard.Clipboard)
         clipboard.setText(text, mode=clipboard.Clipboard)
 
+    def import_streams_from_clipboard(self):
```

Comment:
Perhaps make the import streams a more generic function. Arent we supposed to have a button for importing session? Would be much easier to be able to just connect it to a more generic `import_streams(self, str)`